app/src/main/java/org/fossasia/phimpme/share/SharingActivity.java

```diff
@@ -281,107 +283,111 @@ public void onClick(View view) {
 
     @Override
     public void onItemClick(View childView, final int position) {
-        if (!checkNetwork(this, parent)) return;
-
-        positionShareOption = position;
-        if (sharableAccountsList.get(position) == OTHERS) {
-            triedUploading = true;
-            shareToOthers();
-            return;
-        }
+        ConnectivityManager cm = (ConnectivityManager) context.getSystemService(Context.CONNECTIVITY_SERVICE);
```

Comment:
Instead of doing this, you could have used the function which is already created, checkNetwork().

Reply:
well the snackbar what not working with this code. so i changed it. then i saw the usage of check network function, other than this, it was being used in accounts activity only for checking the network. connectivity manager was already setup so i changed it there. therefore it was not needed anymore.

Reply:
There is no need. In the original code, in the if(!networkCheck()) return; is there. Instead of the return, create a toast there. I tried it, it's working.